### ISSUE TITLE
401 error when trying to update hub on mac

### ISSUE THREAD

**Issue by andrew** [textileio/textile#367]
I get the following error when attempt to update `hub` on mac:

```
$ hub update
> Error! GET https://api.github.com/repos/textileio/textile/releases: 401 Bad credentials []
> Warning! Unable to fetch latest public release.
> v2.0.3
```

**Comment by andrew** [comment]
@sanderpick I still get this error btw

**Comment by sanderpick** [comment]
Oop, I got a little trigger happy closing out issues. @andrewxhill, does this issue ring a bell?

**Comment by andrewxhill** [comment]
@andrew do you get that before even getting this error? https://github.com/textileio/textile/issues/366

**Comment by andrew** [comment]
@andrewxhill yeah, see https://github.com/textileio/textile/issues/366#issuecomment-724146889

**Comment by andrewxhill** [comment]
okay... best current hunch is that the updater lib we rely on is detecting an out-of-date // not authorized token in your environment.

https://github.com/rhysd/go-github-selfupdate/blob/master/selfupdate/updater.go#L55

you can check if you have GITHUB_TOKEN set locally and maybe compare it to your Github account settings to see if it is still valid. if not, maybe just remove it and start a new terminal session.

**Comment by andrew** [comment]
Very strange, I don't have `GITHUB_TOKEN` set in my shell

**Comment by andrewxhill** [comment]
okay, will keep investigating
